Fix for documentation playbook loops

pcraston opened the pull request:
if key is used with lookup parameter that needs to be in quotes as
otherwise it will load "ssh-rsa" "<the key>" "<the comment>" as separate
values and fail with "this module requires key=value arguments"

Commit:
Fix for documentation playbook loops

if key is used with lookup parameter that needs to be in quotes as
otherwise it will load "ssh-rsa" "<the key>" "<the comment>" as separate
values and fail with "this module requires key=value arguments"

mpdehaan commented:
Quotes should also be matched, so this isn't entirely correct.   More likely we want this:

authorized_key: "user={{ item.0.name }} key='{{ lookup('file', item.1) }}'"

Can you update the pull request?

Commit:
Updated Fix for documentation playbook loops

if key is used with lookup parameter needs quotes as
otherwise it will load "ssh-rsa" "<the key>" "<the comment>" as separate
values and fail with "this module requires key=value arguments"

pcraston commented:
OK, I've updated the pull request

mpdehaan commented:
merged, thanks, this will appear when we next push docs!
